CorsixTH/Lua/world.lua

```diff
@@ -878,11 +878,11 @@ end
 -- happens.
 local tick_rates = {
   ["Pause"]              = {0, 1},
-  ["Slowest"]            = {1, 9},
-  ["Slower"]             = {1, 5},
-  ["Normal"]             = {1, 3},
-  ["Max speed"]          = {1, 1},
-  ["And then some more"] = {3, 1},
+  ["Slowest"]            = {1, 28}, -- 1 tick per s
```

Comment:
Not you but would you be able to reword the comment above? I assume "world click" is meant to be "world tick"?

Otherwise I'd like to see how it plays now with the new speeds.